cypress/fixtures/issue-2.json

```diff
@@ -33,8 +33,8 @@
     {
       "id": 5390000409,
       "node_id": "LA_kwDOJWyCVM8AAAABQUThGQ",
-      "url": "https://api.github.com/repos/ubiquity/devpool-directory/labels/Pricing:%20100%20USD",
-      "name": "Pricing: 100 USD",
```

Comment:
We have this changed in the directory backend to not clash with special bot functions. Are you sure this is correct?